// [[Reason: undefined]] ////

//// [tests/cases/conformance/parser/ecmascript2018/asyncGenerators/parser.asyncGenerators.classMethods.es2018.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,21 +1,21 @@
 
 
 //// [/.src/asyncGeneratorGetAccessorIsError.d.ts]
 declare class C23 {
-    get(): any;
+    get(): invalid;
     x(): number;
 }
 
 //// [/.src/asyncGeneratorPropertyIsError.d.ts]
 declare class C25 {
-    x(): any;
+    x(): invalid;
     1: any;
 }
 
 //// [/.src/asyncGeneratorSetAccessorIsError.d.ts]
 declare class C24 {
-    set(): any;
+    set(): invalid;
     x(value: number): void;
 }
 
 //// [/.src/awaitAsTypeIsOk.d.ts]
@@ -88,8 +88,9 @@
 }
 
 //// [/.src/yieldInClassComputedPropertyIsError.d.ts]
 declare class C21 {
+    [yield](): AsyncGenerator<never, void, unknown>;
 }
 
 //// [/.src/yieldInNestedComputedPropertyIsOk.d.ts]
 declare class C22 {
@@ -131,10 +132,13 @@
     f(): AsyncGenerator<number, void, unknown>;
 }
 /// [Errors] ////
 
+asyncGeneratorGetAccessorIsError.ts(2,13): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 asyncGeneratorGetAccessorIsError.ts(2,17): error TS1005: '(' expected.
+asyncGeneratorPropertyIsError.ts(2,13): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 asyncGeneratorPropertyIsError.ts(2,15): error TS1005: '(' expected.
+asyncGeneratorSetAccessorIsError.ts(2,13): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 asyncGeneratorSetAccessorIsError.ts(2,17): error TS1005: '(' expected.
 awaitInParameterInitializerIsError.ts(2,27): error TS2524: 'await' expressions cannot be used in a parameter initializer.
 awaitMissingValueIsError.ts(3,14): error TS1109: Expression expected.
 awaitParameterIsError.ts(2,15): error TS1359: Identifier expected. 'await' is a reserved word that cannot be used here.
@@ -312,26 +316,32 @@
                         ~~~~~~~
 !!! error TS2464: A computed property name must be of type 'string', 'number', 'symbol', or 'any'.
         }
     }
-==== asyncGeneratorGetAccessorIsError.ts (1 errors) ====
+==== asyncGeneratorGetAccessorIsError.ts (2 errors) ====
     class C23 {
         async * get x(): number {
+                ~~~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
                     ~
 !!! error TS1005: '(' expected.
             return 1;
         }
     }
-==== asyncGeneratorSetAccessorIsError.ts (1 errors) ====
+==== asyncGeneratorSetAccessorIsError.ts (2 errors) ====
     class C24 {
         async * set x(value: number): void {
+                ~~~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
                     ~
 !!! error TS1005: '(' expected.
         }
     }
-==== asyncGeneratorPropertyIsError.ts (1 errors) ====
+==== asyncGeneratorPropertyIsError.ts (2 errors) ====
     class C25 {
         async * x = 1: any;
+                ~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
                   ~
 !!! error TS1005: '(' expected.
     }
     
\ No newline at end of file
